src/nyc_trees/apps/home/routes.py

```diff
@@ -19,3 +21,7 @@
 
 training_list_page = route(GET=do(render_template('home/training.html'),
                                   v.training_list_page))
+
+groups_to_follow = route(GET=do(render_template('home/groups_to_follow.html'),
+                                update({'chunk_size': 2}),
```

Comment:
I agree with @kdeloach here.  Adding a new view that calls `group_list` and then adds `chunk_size` to the context should be pretty small and much easier to grok